packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,4 +1,78 @@
-{{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  {{!-- {{yield}} --}}
+
+  {{! Hds-TabList }}
+  <div class="hds-tab-list">
+    <ul class="hds-tab-list__list" role="tablist">
```

Comment:
```suggestion
    <ul class="hds-tabs__tablist" role="tablist">
```

[suggestion] I think the class naming is slightly off. It should at least be `hds-tabs` first since that's the name of the component.

Reply:
I'm going to break this out into separate child components as my next step. I was following naming from @heatherlarsen's CRD: https://docs.google.com/document/d/10OIZG32JaKxAf4uyuDjTU7SGGnwHqabBhPvfHQ8_ruA/edit#bookmark=id.8u2cnqhzum2o